libsolidity/experimental/codegen/IRGenerator.cpp

```diff
@@ -71,8 +71,12 @@ std::string IRGenerator::run(
 	Whiskers t(R"(
 		object "<CreationObject>" {
 			code {
-				codecopy(0, dataoffset("<DeployedObject>"), datasize("<DeployedObject>"))
-				return(0, datasize("<DeployedObject>"))
+				<?eof>
+					returncontract("<DeployedObject>", 0, 0)
+				<!eof>
```

Comment:
I don't fully get it. For EOF in this case we don't  don't use this data related function here. Or maybe I don't understand what is experimental codegen about. :)

Reply:
Moreover on Dialect level these functions are disabled in EOF

Reply:
Also note that this is in ``libsolidity/experimental`` (and thus doesn't really matter) - we wanted to remove the entire thing anyways, didn't we? (Since we moved to an external prototype for it and all)

Reply:
Reverted and `solUnimplementedAssert` added.

Reply:
> I don't fully get it. For EOF in this case we don't don't use this data related function here. Or maybe I don't understand what is experimental codegen about. :)

Sorry, I confused things a bit here. I somehow read it as `return` and assumed the zeros were just placeholders for unimplemented stuff :)

> Moreover on Dialect level these functions are disabled in EOF

Right. Again, my bad :)

Reply:
> we wanted to remove the entire thing anyways, didn't we? (Since we moved to an external prototype for it and all)

I don't now, I don't think you ever mentioned that so far :) So far I was not sure what the status of that experimental implementation was. I.e. whether it will be possible to reuse some of it or whether it has to be rewritten from scratch.

In any case, as long as it's part of the code, it's getting maintained.